.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "cecil"]
 	path = cecil
 	url = https://github.com/jbevain/cecil.git
+[submodule "ILSpy-tests"]
+	path = ILSpy-tests
+	url = https://github.com/icsharpcode/ILSpy-tests
```

Comment:
:memo: I can remove this commit if it is undesirable. I found it much easier to "just work".

Reply:
@dgrunwald What do you think?

Reply:
We use a local branch of this repo to also commit the decompiled code. This allows easily diffing the decompiler results after a test run to see what's changed. But we don't want to push these commits; and we certainly don't want to accidentally commit an updated submodule reference in the main ILSpy repository to point to such a 'decompiled' commit.

Is there a way to avoid 'git commit -a' and GUI tools like TortoiseGit picking up a submodule change if after such local commits? If not, I'd prefer if we didn't add this as a submodule.

Reply:
Actually, there's not really any reason why the test inputs and the decompiled test output needs to be in the same directory/repository.
We could eliminate ILSpy-tests altogether and commit the tests assemblies directly to the main repository (possibly using git LFS to avoid big binaries accumulating in the history); and anyone interested in diffing the outputs can 'git init' a local repo in the decompiled output directory.

Reply:
> Is there a way to avoid 'git commit -a' and GUI tools like TortoiseGit picking up a submodule change if after such local commits?

Yes. We can set `ignore = all` in .gitmodules for this submodule.